conf/drupal/config/views.view.all_documents.yml

```diff
@@ -1351,23 +1351,24 @@ display:
           multi_type: separator
           separator: ', '
           field_api_classes: false
-        field_title:
-          id: field_title
-          table: media__field_title
-          field: field_title
+        view_media_1:
+          id: view_media_1
+          table: media
+          field: view_media
           relationship: none
           group_type: group
           admin_label: ''
-          plugin_id: field
+          entity_type: media
+          plugin_id: entity_link
           label: Title/Permalink
           exclude: false
           alter:
-            alter_text: false
-            text: '<span title="{{ field_file_mime }}">{{ field_title }}</span>'
-            make_link: true
-            path: 'https://www.mass.gov/media/{{ mid }}/download'
-            absolute: false
-            external: true
+            alter_text: true
+            text: 'https://www.mass.gov{{ view_media_1 }}/download'
```

Comment:
I see this is hardcoded to the public URL. Should this be a relative link instead, so it works across different environments? Even so, it would probably be OK to do this as a new ticket, since this is unrelated and how it works on production today.

Reply:
Yeah this is site specific. /download is a special URL in the site. So i think leaving it with hardcoded env URL is fine. At least we used same approach for different views as well.